AUTHORS.rst

```diff
@@ -69,8 +71,8 @@ Some other past or present contributors are:
 Funding
 .......
 
-Alexandre Abraham, Gael Varoquaux, Kamalakar Reddy Daddy, Loïc Estève,
-Mehdi Rahim, Philippe Gervais were paid by the `NiConnect
+`Alexandre Abraham`_, `Gael Varoquaux`_, `Kamalakar Reddy Daddy`_, `Loic Estève`_,
+`Mehdi Rahim`_, `Philippe Gervais`_ were paid by the `NiConnect
 <https://team.inria.fr/parietal/18-2/spatial_patterns/niconnect>`_
```

Comment:
Here we'll have to add the support by HBP for you.
+ the support of Inria for Kshitij.

Reply:
I added these two fundings in 3c51c01
Let me know if you think I should change the wording.
Also, it looks like the link to NiConnect is dead: https://team.inria.fr/parietal/18-2/spatial_patterns/niconnect